fontforge/splineutil.c

```diff
@@ -1451,13 +1451,13 @@ static SplinePointList *SplinePointListCopySpiroSelected1(SplinePointList *spl)
 return( head );
 }
 
-bool SplinePointListCheckSelected1(const SplinePointList *base, bool spiro, bool *allsel) {
+bool SplinePointListCheckSelected1(const SplinePointList *base, bool spiro, bool *allsel, int spiro_cnt_end) {
```

Comment:
As is this is too confusing. My preference is to have this last parameter being a `bool` maybe named `skip_spiro_end` to remove the unused generality. If that isn't acceptable I'm OK with a `spiro_skip_cnt` that must be >= 0 and is subtracted rather than added. 

Otherwise these changes are fine with me.